django/db/models/expressions.py

```diff
@@ -184,6 +184,7 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
            in this query
          * reuse: a set of reusable joins for multijoins
          * summarize: a terminal aggregate clause
+         * for_save: is this expression about to be used in a save or update
```

Comment:
"whether this expression is about to be used..."
(or add a question mark at the end if you prefer the "is" version)